onboard/catkin_ws/src/controls/launch/position_pid.launch

```diff
@@ -7,20 +7,18 @@
         measurements of the max speed the robot can move in x, y, and z directions (local) and set these value 
         accordingly. The units should be meters/sec. Current setup assumes min and max vel are the same.
     /-->
-    <arg name="max_vel_x" default="1" />
-    <arg name="max_vel_y" default="1" />
-    <arg name="max_vel_z" default="1" />
-    <!-- TODO: Determine if these values are correct. Seems like 0.5 was chosen randomly. /-->
-    <arg name="max_vel_roll" default="0.5" />
-    <arg name="max_vel_pitch" default="0.5" />
-    <arg name="max_vel_yaw" default="0.5" />
+    <arg name="max_control_effort" default="1" />
+    <!-- TODO: Determine if orient_max value is correct. Seems like 0.5 was chosen randomly. Currently, this means that
```

Comment:
This was chosen because the robot was turning too quickly compared to how it moves linearly. This is excentuated by abd PID